deploy/olm-catalog/opendatahub/1.4.0/manifests/opendatahub-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,270 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "kfdef.apps.kubeflow.org/v1",
+          "kind": "KfDef",
+          "metadata": {
+            "name": "opendatahub",
+            "namespace": "opendatahub"
+          },
+          "spec": {
+            "applications": [
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odh-common"
+                  }
+                },
+                "name": "odh-common"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "authentication"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odh-dashboard"
+                  }
+                },
+                "name": "odh-dashboard"
+              },
+              {
+                "kustomizeConfig": {
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "odh-notebook-controller"
+                  }
+                },
+                "name": "odh-notebook-controller"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "odh-model-controller"
+                  ],
+                  "repoRef": {
+                    "name": "manifests",
+                    "path": "model-meth"
+                  }
+                },
+                "name": "model-mesh"
+              },
+              {
+                "kustomizeConfig": {
+                  "overlays": [
+                    "metadata-store-mysql",
+                    "ds-pipeline-ui",
+                    "object-store-minio"
```

Comment:
I think we need to add following overlays as given [here](https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/kfctl_openshift_ds-pipelines.yaml#L18-L19)
```
          - integration-odhdashboard
          - default-configs
 ```

Else the minio pods fail to come up and throw error
```
Error: secret "mlpipeline-minio-artifact" not found
```

Reply:
Updated to add `default-configs` and `default-configs` overlays